packages/volto/src/components/manage/Contents/ContentsUploadModal.jsx

```diff
@@ -211,13 +215,13 @@ class ContentsUploadModal extends Component {
           <Header>
             <FormattedMessage id="Upload files" defaultMessage="Upload files" />
           </Header>
+
           <Dimmer active={this.props.request.loading}>
-            <Loader>
-              <FormattedMessage
-                id="Uploading files"
-                defaultMessage="Uploading files"
-              />
-            </Loader>
+            <Progress
+              className="progress-bar"
+              value={this.props.uploadedFiles}
+              total={this.state.totalFiles}
+            >{`Uploading: ${this.props.uploadedFiles} Files Out Of ${this.state.totalFiles}`}</Progress>
```

Comment:
@victorchrollo14 you need to add the message with formatMessage using several attributes.
Here is an example of how it's done within Sharing.jsx
https://github.com/plone/volto/blob/main/packages/volto/src/components/manage/Sharing/Sharing.jsx#L90
https://github.com/plone/volto/blob/main/packages/volto/src/components/manage/Sharing/Sharing.jsx#L434
and how it looks when the message is extracted
https://github.com/plone/volto/blob/main/packages/volto/locales/en/LC_MESSAGES/volto.po#L407

We do not want the message to be hardcoded to english since Volto is a multi language framework.

Reply:
yeah @stevepiercy mentioned that, I'll add that up.